storm/src/main/storm/mesos/util/PrettyProtobuf.java

```diff
@@ -129,13 +129,32 @@ public static String taskInfoListToString(List<TaskInfo> tasks) {
    * TODO:(erikdw): figure out a design better that removes the need
```

Comment:
can you fix my stupid typo of "design better"?  (i.e., should be "better design")